supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/VpcAndSubnetworkProjectLabelValues.scala

```diff
@@ -0,0 +1,3 @@
+package cromwell.backend.google.pipelines.common
+
+final case class VpcAndSubnetworkProjectLabelValues(vpcName: Option[String], subnetName: Option[String])
```

Comment:
This case class model allows for invalid state to be created (ie `VpcAndSubnetworkProjectLabelValues(None, Some(...))` - and therefore invalid state has to be handled. I think it would be neater overall to replace fields or parameters which currently look like:
```
final case class VpcAndSubnetworkProjectLabelValues(vpcName: Option[String], subnetName: Option[String])
...
val subnetworkConfig: VpcAndSubnetworkProjectLabelValues = ???
def foo(subnetworkConfig: VpcAndSubnetworkProjectLabelValues) = ???
```

With a type model which doesn't allow invalid states, like:
```
final case class VpcRequirementAndSubnetwork(vpcName: String, subnetName: Option[String])
...
val vpcAndSubnetRequirement: Option[VpcAndSubnetworkProjectLabelValues] = ???
def foo(vpcAndSubnetRequirement: Option[VpcAndSubnetworkProjectLabelValues]) = ???
```